src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,311 @@ paths:
+
+        responses:
+          200:
+            description: "The alert status has been updated successfully."
+            schema:
+              $ref: '#/definitions/alert_patched'
+          400:
+            description: "Bad Request. The payload may be malformed, or an invalid alertType may have been provided in URL."
+            schema:
+              $ref: '#/definitions/alert_patched_bad_request_error_message'
+          401:
+            description: "Unauthorized"
+          404:
+            description: "No alert found for the assetId and alertType provided."
+            schema:
+              $ref: '#/definitions/alert_patched_not_found_error_message'
+          500:
+            description: "Internal Server Error"
+    delete:
+      operationId: "alertDelete"
+      tags:
+        - "Alert subscriptions"
+      summary: Delete the alert for a particular query and alert type
+      description: This request deletes the alert for a particular query or schedule ID and alert type.
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/User-Agent'
+
+        - name: assetId
+          description: A query ID or a schedule ID that you want to delete the alert for.
+          required: true
+          in: path
+          type: string
+
+        - name: alertType
+          description: The state of query execution that triggers the alert to be deleted. The alert type is one of three values (start, success, failure).
+          required: true
+          in: path
+          type: string
+
+      responses:
+        200:
+          description: "Alert deleted successfully."
+          schema:
+            $ref: '#/definitions/alert_deleted'
+        400:
+          description: "Bad Request. An invalid alertType was provided in the URL."
+          schema:
+            $ref: '#/definitions/alert_deleted_bad_request_error_message'
+        401:
+          description: "Unauthorized"
+        404:
+          description: "No alert was found for the given assetId and alertType."
+          schema:
+            $ref: '#/definitions/alert_deleted_not_found_error_message'
+        500:
+          description: "Internal Server Error."
+  '/alert-subscriptions/user-subscriptions/{emailID}':
+    get:
+      operationId: "listAllAlertsSubscribedToByEmailId"
+      tags:
+        - "Alert subscriptions"
+      summary: List information on all alerts that a user is subscribed to
+      description: This request searches all alerts for the email ID provided and returns a list of all the alerts that a user is subscribed to. The returned information includes the alert name, IDs, status, alert type, and notification channels. 
```

Comment:
"all"? 

"for the email ID provided" Provided where?

Can a single alert have multiple IDs?